apps/antalmanac/src/components/RightPane/RightPaneStore.ts

```diff
@@ -80,17 +87,40 @@ class RightPaneStore extends EventEmitter {
         this.emit('formDataChange');
     };
 
+    storePrevFormData = () => {
+        this.prevFormData = structuredClone(this.formData);
+        this.emit('formDataChange');
+    };
+
+    restorePrevFormData = () => {
+        if (!this.prevFormData) return;
```

Comment:
nit: always brace your conditionals
```suggestion
        if (!this.prevFormData) {
            return;
        }
```